components/profile/appAvatar/index.test.jsx

```diff
@@ -46,11 +43,3 @@ describe("App Avatar", () => {
     expect(asFragment()).toMatchSnapshot();
   });
 });
-
-describe("setupErrorComponent", () => {
```

Comment:
This test case is really testing internals of the AppAvatar component — it'd be better to find a way to inject an error state into the component. Additionally, it's worth looking into whether `errorComponent` can just be defined as a stand-alone component, rather than a component factory.